core/services/blockhashstore/delegate.go

```diff
@@ -91,7 +96,13 @@ func (d *Delegate) ServicesForSpec(jb job.Job) ([]job.ServiceCtx, error) {
 
 			return nil, errors.Wrap(err, "building V1 coordinator")
 		}
-		coordinators = append(coordinators, NewV1Coordinator(c))
+
+		var coord *V1Coordinator
+		coord, err = NewV1Coordinator(c, lp)
```

Comment:
```suggestion
		coord, err := NewV1Coordinator(c, lp)
```

Reply:
This causes "err" to be shadowed which throws for our linter.

Reply:
I guess can do `coord, err2 := ...` then but I'm not sure what's uglier now 😄 .

Reply:
```
There are a number of ways to deal with this potential problem. The most direct is to avoid := by declaring err in a separate var declaration:

var cwd string

func init() {
    var err error
    cwd, err = os.Getwd()
    if err != nil {
        log.Fatalf('os.Getwd failed: %v', err)
    }
}
```
Straight from go programming language book. Would just go with the explicit declaration first rather than renaming err.